ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-master-presubmits.yaml

```diff
@@ -661,6 +661,71 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp-operator,?($|\s.*)
+  - agent: kubernetes
```

Comment:
please make this job optional for now as in https://github.com/openshift/release/blob/dc19eabbb6eab81ea10e5ad185c77a9d36857972/ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-master-presubmits.yaml#L385

Reply:
Done - line #677